test/__fixtures__/workspace/select-error2.txt

```diff
@@ -0,0 +1,5 @@
+You have not selected any the of following - Project
+You are currently in:
```

Comment:
doesn't make sense. error doesn't show Org, but Org config shows is 'no org selected'